marketaccess/templates/marketaccess/report_barrier_form/step-others.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'marketaccess/report_barrier_form/base.html' %}
+
+{% load static from staticfiles %}
+
+{% block extra_contents %}
+    <p>Report a trade barrier</p>
+    <p class="bold-small"> Step 3 of 4 </p>
```

Comment:
`Step x of y` could be `{{ wizard.steps.step1 }} of {{ wizard.steps.count }}`

that's beneficial in case you add new steps, remove steps. change order etc.